gateway-release/pom.xml

```diff
@@ -144,6 +144,14 @@
                                             <outputDirectory>${project.build.directory}</outputDirectory>
                                             <includes>applications/**/*</includes>
                                         </artifactItem>
+                                        <artifactItem>
```

Comment:
Need to see what this looks like in the final tar.gz to make sure this is valid packaging since `gateway-release` basically builds the final apache builds.

Reply:
I'll add this info to the PR description later today.

Reply:
Done